src/ImageRunModal.jsx

```diff
@@ -1089,6 +1165,24 @@
                 </Tabs>
             </Form>
         );
+
+        const createRunText = this.props.prefill ? _("Clone and run") : _("Create and run");
+        const createText = this.props.prefill ? _("Clone") : _("Create");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-110256-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)